store/index.ts

```diff
@@ -0,0 +1,21 @@
+import { ArticleInterface } from "~/utils/interfaces"
+
+export const state = () => ({
+    selectedArticle: {} as Record<string, unknown>
+})
+
+export const getters = {
+    getSelectedArticle(state: Record<string, unknown>) {
+        return state.selectedArticle
+    }
+}
+
+export const mutations = {
+    SET_SELECTED_ARTICLES(state : Record<string, unknown>, data: { articleData: ArticleInterface }) {
+        state.selectedArticle = Object.assign({}, data.articleData)
+    }
+}
+
+export const actions = {
+    // actions
+}
```

Comment:
please remove this as there are no actions to define

Reply:
Fixed.